packages/medusa-react/src/hooks/admin/payment-collections/queries.ts

```diff
@@ -0,0 +1,32 @@
+import { queryKeysFactory } from "../../utils"
+import { StorePaymentCollectionRes } from "@medusajs/medusa"
+import { useQuery } from "react-query"
+import { useMedusa } from "../../../contexts"
+import { UseQueryOptionsWrapper } from "../../../types"
+import { Response } from "@medusajs/medusa-js"
+
+const PAYMENT_COLLECTION_QUERY_KEY = `paymentCollection` as const
+
+export const paymentCollectionQueryKeys = queryKeysFactory<
+  typeof PAYMENT_COLLECTION_QUERY_KEY
+>(PAYMENT_COLLECTION_QUERY_KEY)
+
+type OrderQueryKey = typeof paymentCollectionQueryKeys
+
+export const usePaymentCollection = (
```

Comment:
todo: Missing domain from name
```suggestion
export const useAdminPaymentCollection = (
```